crypto/verifier.go

```diff
@@ -31,8 +31,13 @@ import (
 	"github.com/google/trillian/crypto/sigpb"
 )
 
-// ErrVerify occurs whenever signature verification fails.
-var ErrVerify = errors.New("signature verification failed")
+var (
+	errVerify = errors.New("signature verification failed")
```

Comment:
One can use type assertions to distinguish between types defined with `type newError ...`  
All errors from `error.New` or `fmt.Errorf` return the standard error type `stringError` which is better distinguished with an equality operator that simply compares the `Error()` strings.